packages/variant-theme/src/ThemeGenerator.ts

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable */
+/* tslint-disable */
+//@ts-nocheck
+import {
+  getColorFromString,
+  IColor,
+  updateA
+} from '@fluentui/react';
+import { BrandVariants, Theme } from '@fluentui/react-components';
+import { VariantThemeType } from '@fluentui/scheme-utilities';
```

Comment:
always specify kind of import in order to improve tree-shakibility

```suggestion
import type { VariantThemeType } from '@fluentui/scheme-utilities';
```

Reply:
I also use the VariantThemeType.None and something like it in the switch case. So I don't import it as type